### DISCUSSION TITLE
gh-pages is not working

### DISCUSSION THREAD

**Discussion by Gokul2k03** [community/community#51382] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I got the message published in the terminal but when I open the link it is not working. 
https://gokul2k03.github.io/task2
This is my link need to be published but it shows 404 page not found

**Comment by CodeMaster7000** [comment]
Hi @Gokul2k03,

It seems like a temporary GitHub servers problem. This is the case as GitHub Pages are running at "degraded" performance. This issue has been reported [here](https://www.githubstatus.com/incidents/z3c6q056q332) and is under investigation.

You can check the [GitHub status page](https://www.githubstatus.com/) to get real-time information on current performance. There is also the option of subscribing for updates.

**Comment by radar-blip** [comment]
https://gokul2k03.github.io/task2/ is up and running now.

**Reply by CodeMaster7000** [reply]
Yes, it seems like GitHub has resolved the issue.
